packages/eas-cli/src/project/projectUtils.ts

```diff
@@ -97,3 +100,47 @@ export function getProjectConfigDescription(projectDir: string): string {
   }
   return 'app.config.js/app.json';
 }
+
+export async function getReleaseByNameAsync({
+  appId,
+  releaseName,
+}: {
+  appId: string;
+  releaseName: string;
+}): Promise<UpdateRelease> {
```

Comment:
What's returned by this function if the release doesn't exist (e.g. there's a typo)? Or does it throw an error? I think this error should be handled in a documented way, for example throw an error like `error.code === 'RELEASE_NOT_FOUND` or similar. (Perhaps the error returned by the API is self descriptive? In that case we can just let it bubble up.)

Reply:
The error is pretty self descriptive, for example:

<img width="1543" alt="Screen Shot 2020-12-08 at 8 11 56 AM" src="https://user-images.githubusercontent.com/1220444/101508897-0b403180-392d-11eb-95ac-44cd02cefb2f.png">

Reply:
That's pretty good, but we should not show the `CombinedError: [GraphQL] ` cruft to the user. Additionally `@owner/slug` might be nicer to show as the app identifier.

Reply:
OK, it seems like the the standard way of doing that is to handle it at the call site. I can do that in the upcoming `release:publish` PR.

Reply:
Thinking about this more, it seems like the graphQL errors should all be formatted here: https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/graphql/client.ts#L44

Regarding the `@owner/slug` point, I will put up a PR server side to fix that.